tests/playwright/pages/profile.spec.ts

```diff
@@ -43,7 +44,7 @@ test.describe("Admin profile", () => {
   );
 
   test(
-    "Is accessible on all device types for admin user",
+    "Profile page is accessible on all device types for admin user",
     { tag: "@admin" },
```

Comment:
not related to this PR, but I just noticed that a few of these tests have tags, though not all. Is this a feature we want to apply to all? if not I would suggest removing these, unless they are providing some other benefit